scripts/migrate-to-bcd.ts

```diff
@@ -0,0 +1,90 @@
+import assert from 'node:assert';
+import fs from 'node:fs';
+import path from 'node:path';
+import {fileURLToPath} from "node:url";
+import { fdir } from 'fdir';
+import { features } from '../index.js';
+
+export const BCD_PATH = process.env.BCD_PATH
+  ? path.resolve(process.env.BCD_PATH)
+  : fileURLToPath(new URL("../../browser-compat-data", import.meta.url));
+
+// Map from api.CoolThing.something to cool-thing
+const bcdToFeature = new Map();
+
+// Set of the first part of the BCD path encountered.
+const bcdDirs = new Set();
+
+for (const [feature, {compat_features}] of Object.entries(features)) {
+    if (!compat_features) {
+        continue;
+    }
+    for (const key of compat_features) {
+        assert(!bcdToFeature.has(key), `${key} is used in compat_features twice`);
+        bcdToFeature.set(key, feature);
+        bcdDirs.add(key.split('.')[0]);
+    }
+}
+
+const bcdJsons = new fdir()
+    .withBasePath()
+    .filter((fp) => {
+        const dir = path.relative(BCD_PATH, fp).split(path.sep)[0];
+        return bcdDirs.has(dir);
+    })
+    .filter((fp) => fp.endsWith('.json'))
+    .crawl(BCD_PATH)
+    .sync();
+
+const lookup = (root, key) => {
```

Comment:
Style nit: we've been mostly using `function` in scripts here. Would you mind re-doing this as a plain function?